### PR TITLE
gm-editor: Small usability tweaks

### PR DESCRIPTION
**Pull request by Cubittus**
Three small inprovements:
- Don't clobber previous window frame when opening a new one
- Enable jumping to matinfos
- Auto-fit the key column

### CODE REVIEW

File: gui/gm-editor.lua [hunk 1]
```diff
@@ -72,7 +72,7 @@ end
 
 GmEditorUi = defclass(GmEditorUi, widgets.Window)
 GmEditorUi.ATTRS{
-    frame=config.data,
+    frame={ w=config.data.w, h=config.data.h, l=config.data.l, r=config.data.r, t=config.data.t, b=config.data.b },
```
Comment (myk002) [inline gui/gm-editor.lua:75]:
```suggestion
    frame=copyall(config.data),
```

**Review by myk002 — COMMENTED** [review]
could you add changelog entries in `changelog.txt` (under Misc Improvements)?

File: changelog.txt [hunk 2]
```diff
@@ -23,6 +23,7 @@ that repo.
 - `suspendmanager`: does not suspend non blocking jobs such as floor bars or bridges anymore
 - `suspendmanager`: fix occasional bad identification of buildingplan jobs
 - `warn-starving`: no longer warns for enemy and neutral units
+- `gui/gm-editor`: no longer nudges last open window when opening a new one
```
Comment (myk002) [inline changelog.txt:26]: this file looks out of date -- after yesterday's release, these lines are now in the "50.07-beta2" section. Could you merge current HEAD or rebase, then move these new lines to the new "Future" section?

Comment (Cubittus) [reply changelog.txt:26]: Whoops! - will do

**Review by myk002 — APPROVED** [review]

**Pull request merged**